shared/profile/container.js

```diff
@@ -68,9 +41,8 @@ class ProfileContainer extends PureComponent<void, EitherProps<Props>, State> {
 
     return <Profile
       {...props}
-      onAvatarLoaded={this._onAvatarLoaded}
-      followers={this.state.avatarLoaded ? props.followers : null}
-      following={this.state.avatarLoaded ? props.following : null} />
+      followers={props.followers}
```

Comment:
yes. We needed to do this cause we were being rate limited by the api server but that whole issue has been resolved so the image loading story is a lot better. I think this pattern is also semi-dangerous. so i removed it